ui/desktop/app/styles/app.scss

```diff
@@ -292,3 +292,38 @@
   margin: sizing.rems(l) 0 0 0;
   text-align: center;
 }
+
+.credentials-card {
+  border: sizing.rems(xxxxs) solid var(--ui-gray-subtler-3);
+  display: flex;
+  flex-direction: column;
+  margin-bottom: sizing.rems(m);
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  .credentials-card-header {
+    border-bottom: sizing.rems(xxxxs) solid var(--ui-gray-subtler-3);
+    display: flex;
+    flex-direction: row;
+    padding: sizing.rems(s);
+
+    div {
+      display: flex;
+      flex-grow: 1;
```

Comment:
This makes the header ready to receive more than 1 `<div>` (as item) and rearrange itself. `div` it is very specific, if you like, I can make it more generic creating a class for it, something like `credentials-card-header-item`